docs/reference/platforms/macOS/app.rst

```diff
@@ -16,118 +16,23 @@ A macOS ``.app`` bundle is a collection of directory with a specific layout,
 and with some key metadata. If this structure and metadata exists, macOS treats
 the folder as an executable file, giving it an icon.
 
-``.app`` bundles can be copied around as if they are a single file. They can
-also be compressed to reduce their size for transport.
-
-By default, apps will be both signed and notarized when they are packaged.
-
-Packaging format
-================
-
-Briefcase supports three packaging formats for a macOS app:
-
-1. A DMG that contains the ``.app`` bundle (using ``briefcase package macOS -p dmg``).
-2. A zipped ``.app`` folder (using ``briefcase package macOS -p zip``).
-3. A ``.pkg`` installer (using ``briefcase package macOS -p pkg``).
-
-``.pkg`` is the *required* format for console apps. ``.dmg`` is the
-default format for GUI apps.
-
-Icon format
-===========
-
-macOS ``.app`` bundles use ``.icns`` format icons.
-
-macOS ``.app`` bundles do not support splash screens or installer images.
+An ``.app`` bundle is the default Briefcase output format when running on macOS.
+However, you can explicitly specify the use of the ``.app`` bundle backend by using
+``briefcase <command> macOS app`` when invoking Briefcase.
```

Comment:
"when invoking Briefcase" seems redundant.

There are 4 instances of this in the PR.